templates/zerver/keyboard_shortcuts.html

```diff
@@ -134,6 +134,14 @@
           <td class="hotkey">?</td>
           <td class="definition">{% trans %}Show these keyboard shortcuts{% endtrans %}</td>
         </tr>
+        <tr>
+          <td class="hotkey">:</td>
+          <td class="definition">{% trans %}Open reactions popover menu{% endtrans %}</td>
+        </tr>
+        <tr>
+          <td class="hotkey">+</td>
+          <td class="definition">{% trans %}React to selected message with +1{% endtrans %}</td>
```

Comment:
Might want to add `<img alt=":thumbs_up:" class="emoji" src="/static/generated/emoji/images/emoji/unicode/1f44d.png" title=":thumbs_up:">` in place of `+1`, or include of them within parentheses.